v0/destinations/freshmarketer/data/FRESHMARKETERIdentifyConfig.json

```diff
@@ -0,0 +1,194 @@
+[
+  {
+    "destKey": "emails",
+    "sourceKeys": "email",
+    "sourceFromGenericMap": true,
+    "required": true
+  },
+  {
+    "destKey": "first_name",
+    "sourceKeys": "firstName",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "last_name",
+    "sourceKeys": "lastName",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "subscription_status",
+    "sourceKeys": [
+      "traits.subscriptionStatus",
+      "context.traits.subscriptionStatus"
+    ],
+    "required": false
+  },
+  {
+    "destKey": "job_title",
+    "sourceKeys": [
+      "traits.job_title",
+      "traits.jobTitle",
+      "context.traits.job_title",
+      "context.traits.jobTitle"
+    ],
+    "required": false
+  },
+  {
+    "destKey": "work_number",
+    "sourceKeys": "phone",
+    "sourceFromGenericMap": true,
+    "required": false
```

Comment:
We can't use phone as work number.

Reply:
In Segment docs, they have mapped it to the work number.
ref: https://segment.com/docs/connections/destinations/catalog/freshsales/